From: Kunal Mehta <legoktm@member.fsf.org>
Date: Tue, 25 Sep 2018 21:44:50 -0700
Subject: Document removal of CologneBlue & Modern from the tarball

Worth noting as a pre-upgrade thing, as trying to run the updater will fail
unless it's disabled or downloaded separately.

Reported by <https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=909589>.

Forwarded: https://gerrit.wikimedia.org/r/c/mediawiki/core/+/462846
Change-Id: I5ea6dc65d683dfceaa25104ca59f2f56c1ab1884
---
 RELEASE-NOTES-1.31 | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/RELEASE-NOTES-1.31 b/RELEASE-NOTES-1.31
index b0da15a..7bdb816 100644
--- a/RELEASE-NOTES-1.31
+++ b/RELEASE-NOTES-1.31
@@ -55,6 +55,10 @@ This is a security and maintenance release of the MediaWiki 1.31 branch.
   to the ar_text and ar_flags columns of the archive table or make those
   columns nullable before upgrading to MediaWiki 1.31.
   maintenance/archives/patch-nullable-ar_text.sql shows how to do this for MySQL.
+* The CologneBlue and Modern skins are no longer bundled with the tarball. You
+  will need to remove the wfLoadSkin() calls from your LocalSettings.php or
+  download them separately
+  (<https://www.mediawiki.org/wiki/Special:SkinDistributor>).
 
 === Configuration changes in 1.31 ===
 * $wgEnableAPI and $wgEnableWriteAPI are now deprecated and will be removed in
